x-pack/legacy/plugins/alerting/README.md

```diff
@@ -101,8 +101,8 @@ server.plugins.alerting.registerType({
 				cpuUsage: currentCpuUsage,
 			});
 
-			// 'default' refers to a group of actions to fire, see 'actions' in create alert section
-			alertInstance.fire('default', {
+			// 'default' refers to a group of actions to enqueue to be fired, see 'actions' in create alert section
```

Comment:
There's a few places in this file here and below that still use the old function names and terminology. We could remove any reference to `fire` and `enqueue` and stick with `schedule` and `execute`, thoughts?